src/mono/browser/runtime/loader/assets.ts

```diff
@@ -450,7 +475,17 @@ async function start_asset_download_with_throttle (asset: AssetEntryInternal): P
         if (skipBuffer) {
             return asset;
         }
-        asset.buffer = await response.arrayBuffer();
+        if (
+            (asset.behavior !== "vfs") && // vfs needs an arraybuffer so just use .arrayBuffer()
+            ("body" in response) && response.body && // in some cases response.body can be missing or null
+            asset.knownLength
+        ) {
+            asset.stream = [response.body!, asset.knownLength];
```

Comment:
Should we detect that the stream is supported, before we claim it's a stream ?

Reply:
If response.body exists it should be a supported stream